### PR TITLE
Fix a bug: picture get no width height

### PR DESCRIPTION
**Pull request by 401885064**
In China, some mobile phone pictures returned to the picture is not to get the width and height, so we need to solve this bug

### CODE REVIEW

File: ucrop/src/main/java/com/yalantis/ucrop/task/BitmapLoadTask.java [hunk 1]
```diff
@@ -110,7 +110,23 @@ protected BitmapWorkerResult doInBackground(Void... params) {
         options.inJustDecodeBounds = true;
         BitmapFactory.decodeFileDescriptor(fileDescriptor, null, options);
         if (options.outWidth == -1 || options.outHeight == -1) {
-            return new BitmapWorkerResult(new IllegalArgumentException("Bounds for bitmap could not be retrieved from the Uri: [" + mInputUri + "]"));
+            //return new BitmapWorkerResult(new IllegalArgumentException("Bounds for bitmap could not be retrieved from the Uri: [" + mInputUri + "]"));
+			
+			try {
```
Comment (Cool04ek) [inline ucrop/src/main/java/com/yalantis/ucrop/task/BitmapLoadTask.java]: please, format code properly

File: ucrop/src/main/java/com/yalantis/ucrop/task/BitmapLoadTask.java [hunk 2]
```diff
@@ -110,7 +110,23 @@ protected BitmapWorkerResult doInBackground(Void... params) {
...
+                ExifInterface exifInterface = new ExifInterface(mInputUri.getPath());
+                int height = exifInterface.getAttributeInt(ExifInterface.TAG_IMAGE_LENGTH, ExifInterface.ORIENTATION_NORMAL);//获取图片的高度
+                int width = exifInterface.getAttributeInt(ExifInterface.TAG_IMAGE_WIDTH, ExifInterface.ORIENTATION_NORMAL);//获取图片的宽度
+                Log.d(TAG, "exif height: " + height);
+                Log.d(TAG, "exif width: " + width);
+                options.outWidth = width;
+                options.outHeight = height;
+
+                mRequiredWidth=width/3;
```
Comment (Cool04ek) [inline ucrop/src/main/java/com/yalantis/ucrop/task/BitmapLoadTask.java]: why do you need to change required height and width?

Comment (401885064) [inline ucrop/src/main/java/com/yalantis/ucrop/task/BitmapLoadTask.java]: I not  want to change height and width , is do not get height and width, is null, is throw exception.

Comment (Cool04ek) [inline ucrop/src/main/java/com/yalantis/ucrop/task/BitmapLoadTask.java]: I mean why do you need to set mRequiredHeight = height / 3. What does the "3" means?

Comment (401885064) [inline ucrop/src/main/java/com/yalantis/ucrop/task/BitmapLoadTask.java]: oh，is my request！  no things..

**Pull request closed without merging**